apps/chat/src/components/Chat/Publish/PublicationHandler/ReviewRowItems/PublicationFolderRow.tsx

```diff
@@ -79,30 +80,41 @@ export const PublicationFolderRow = <T extends PublicationReviewItem>({
     PublicationSelectors.selectSelectedItemsToApprove,
   );
 
+  const folderEditState = useAppSelector(
+    PublicationSelectors.selectFoldersEditState,
+  );
+
+  const getValidationErrors = useCallback(
```

Comment:
guess useMemo suits better here
```suggestion
  const validationErrors = useMemo(
```